setup/MSBuild.swixproj

```diff
@@ -45,5 +45,7 @@
     </PropertyGroup>
   </Target>
 
+  <!-- MicroBuild plugin fails if GetNativeManifest is not found for some reason -->
+  <Target Name="GetNativeManifest" />
```

Comment:
This is pretty common behavior--I think it's just from using `ProjectReference` from common targets.